bin/mix.bat

```diff
@@ -1,2 +1,2 @@
 @if defined ELIXIR_CLI_ECHO (@echo on) else  (@echo off)
-call "%~dp0\elixir.bat" -e Mix.start -e Mix.CLI.main %*
+call "%~dp0\elixir.bat" "%~dp0\mix.exs" %*
```

Comment:
Can you just run `"%~dp0\mix"` instead? The `mix.exs` file you created is exactly the same as `bin/mix`. :)